### DISCUSSION TITLE
Release Notes CovPass and CovPassCheck Version 1.16

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-android#171] [Announcements]
📢 ***Release Notes CovPass and CovPassCheck Version 1.16*** 📢   

**Main Features CovPass:**

- Neutral Start Screen: The start screen has been visually adjusted so that it is no longer immediately obvious what type of certificate it is. This protects users' personal data and promotes the correct use of the CovPassCheck app.

- 2/1 Recognition: The apps now will detect 2/1 vaccinations (also 3/1, 4/1 etc.) as booster-vaccinations within the CovPass. An adjustment for CovPassCheck will be made using the digital EU Business Rules so that the new scheme will be not only accept but lead to immediately valid certificates etc.

  
**Planned Main Features CovPassCheck:**
•	No new main features are included in this version. A requested adaption for 2G+ concepts etc. is planned for the release 1.17.  Please also check the new announcement “Information about the upcoming version 1.17” for further information.

  
**Bug-Fixes:**
- Auto-Focus: Android-Devices that had troubles with the auto-focus while scanning certificates should now be able to scan certificates properly
- Remove-Onboarding on Updates: For Android the onboarding process will not be shown after an update. However new data-privacy statements need to accept during first startup after an update.
- Reference to full vaccination: With the neutral screen feature the information about the date of completed vaccination on the start screen will disappear. With that we will no longer reference the date of booster as the date of full immunization.
- Update on the rule sets: For IOS up-to-date local rule sets will be stored. With that boosters will be immediately valid after first startup. No manual update of the rule-sets is required.
- Fix on Flip- and Fold-Device Crashes: Flip- and Fold-Devices will no longer crash during scan. With that certificates can now be stored with the latest version of the CovPass and validated with the latest version of the CovPassCheck. Further UI adjustments are planned for upcoming releases.
  
❗ **Attention:**  Please also check the new announcement “Information about the upcoming version 1.17” for further information about upcoming features and bugfixes. Besides the upcoming features we are planning to adapt the current EU Rules to the new requirements. These updates are mostly independent of our release schedules. We will inform you about further updates.

Thank you all for your feedback, ideas and critics!   

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by Ein-Tim** [comment]
Same as #169?

**Reply by alexcimander** [reply]
yes! We managed to get all of our scheduled features and fixes into the app. 
What is your opinion here: Does it make sense to have an "information about upcoming releases" and "release notes" of the actual release? I guess with both notes we can make more transparent what really worked out or what maybe changed due to the dynamic pandemic 🤔 👍
 
Besides that I guess there is room for improvement when it comes to plattform specific features and fixes. For 1.17 I want to split between IOS and Android related fixes (fyi @timokoenig)

**Reply by Ein-Tim** [reply]
Okay, yes @alexcimander, this makes sense!

**Reply by timokoenig** [reply]
@alexcimander that's really great to hear!

The idea is good. Another solution would be a general repository where all tickets belong that handle both platforms. This would reduce the duplicates and can also act as a good place for release notes. Reason for that is that people who don't watch the other repository (Android or iOS) will not get notifies about any release discussions and tickets that would affect both platforms. Just an idea that could be discussed

**Comment by vaubaehn** [comment]
Hi @alexcimander and @timokoenig ,
regarding the 2/1, 3/1... recognition, in line with the changed EU notation, that's great.
But with the latest changes in the German vaccination schema, now J&J vaccinations need 2 jabs for basic immunization, and a 3rd to be boostered.
So, theoretically, Germany could continue to use the 1/1, 2/2, 3/3... notation for J&J, and it will be clear that everything with `dn>2` is a booster...
Could you please reveal, how the notation for J&J vaccinations/boosters is planned from February on (via issuance portal)?
That information could be helpful for us to support community via social media/GitHub.
Thank you!

**Comment by Crossfade14** [comment]
Unfortunately the app does not show the 2/1 status for my booster after the update. I have an certificate from June showing 1/1 which is the old Biontech shot for recovered persons. The Moderna booster from December is still shown as 2/2. Do I have to do anything else to make ist work? Thank you very much!

**Reply by Ein-Tim** [reply]
@Crossfade14 As far as I know pharmacies will start to issue certificates with the changed numbers on 01.02.2022
